indium-debugger.el

```diff
@@ -351,7 +351,20 @@ Return nil if no local file can be found."
 
 (defun indium-debugger-get-current-scopes ()
   "Return the scope of the current stack frame."
-  (map-elt (indium-debugger-current-frame) 'scope-chain))
+  (seq-sort-by
+   (lambda (scope)
+     ;; Possible protocol values: global, local, with, closure, catch, block, script, eval, module.
+     ;; We also add "this" to the list.
+     (pcase (map-elt scope 'type)
```

Comment:
That's too specific to Chrome's debugger protocol to be in `indium-debugger.el`.